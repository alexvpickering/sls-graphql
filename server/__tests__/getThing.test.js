import axios from "axios";
import gql from "graphql-tag";

it("gets a thing", () => {
  const variables = { id: "1" };
  const query = gql`
    query GetThing($id: ID!) {
      getThing(id: $id) {
        two
      }
    }
  `;

  return axios
    .post("http://localhost:4000/graphql", {
      query,
      variables
    })
    .then(res => {
      expect(res.status).toBe(200);
      expect(res.data.data).toMatchSnapshot();
    });
});
