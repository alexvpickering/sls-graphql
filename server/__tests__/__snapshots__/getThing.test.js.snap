// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gets a thing 1`] = `
Object {
  "getThing": Object {
    "two": "thing 2",
  },
}
`;
